inspirehep/modules/records/api.py

```diff
@@ -24,45 +24,92 @@
 
 from __future__ import absolute_import, division, print_function
 
+import copy
 from datetime import datetime
 
 import arrow
 from elasticsearch.exceptions import NotFoundError
 from flask import current_app
 
 from inspire_schemas.api import validate
+from inspire_schemas.builders import LiteratureBuilder
 from invenio_files_rest.models import Bucket
 from invenio_pidstore.errors import PIDDoesNotExistError
 from invenio_pidstore.models import PersistentIdentifier
 from invenio_records_files.api import Record
 from invenio_db import db
 
+from inspirehep.modules.records.utils import open_url_or_path
 from inspirehep.utils.record_getter import (
     RecordGetterError,
     get_es_record_by_uuid
 )
 
 
+MAX_UNIQUE_KEY_COUNT = 50000
+
+
 class InspireRecord(Record):
 
     """Record class that fetches records from DataBase."""
 
+    @classmethod
+    def create(cls, *args, **kwargs):
+        """
+
+        Args:
```

Comment:
Napoleon includes a specific section for kwargs, named not surprisingly `Keyword Args`: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/index.html#sections.

Reply:
Good to know 👍